### PR TITLE
A very sad PR to update the current team

### PR DESCRIPTION
**Pull request by chitrakotwani**

### CODE REVIEW

**Review by danspndl — APPROVED** [review]
😢

File: README.md [hunk 1]
```diff
@@ -71,9 +70,8 @@ The rest of the iOS Engineers work in the following squads:
...
       <tr><td>Nikolay Marinov</td><td><a href='https://github.com/nikolay-bab'>@nikolay-bab</a></td><td></td></tr>
-    <tr><td rowspan='3' valign='top'><strong>Native Apps Platform</strong><br/>Engineering work like tooling, CI and development processes.</td>
-      <td>David Rodrigues</td><td><a href='https://github.com/dmcrodrigues'>@dmcrodrigues</a></td><td><a href='https://twitter.com/dmcrodrigues'>@dmcrodrigues</a></td></tr>
-      <tr><td>Ilya Puchka</td><td><a href='https://github.com/ilyapuchka'>@ilyapuchka</a></td><td><a href='https://twitter.com/ilyapuchka'>@ilyapuchka</a></td></tr>
+    <tr><td rowspan='2' valign='top'><strong>Native Apps Platform</strong><br/>Engineering work like tooling, CI and development processes.</td>
+      <td>Ilya Puchka</td><td><a href='https://github.com/ilyapuchka'>@ilyapuchka</a></td><td><a href='https://twitter.com/ilyapuchka'>@ilyapuchka</a></td></tr>
       <tr><td>Olivier Halligon</td><td><a href='https://github.com/AliSoftware'>@AliSoftware</a></td><td><a href='https://twitter.com/aligatr'>@aligatr</a></td></tr>
     <tr><td rowspan='2' valign='top'><strong>Prescriptions</strong><br/>Prescriptions functionality.</td>
```
Comment (konrad-em) [inline README.md:76]:
```suggestion
    <tr><td rowspan='2' valign='top'><strong>Real Time Matching</strong><br/>Prescriptions functionality.</td>
```

**Review by konrad-em — APPROVED** [review]
I just added one suggestion for a name update of my squad to go along with other updates.

**Review by mluisbrown — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chitrakotwani) [reply README.md:76]: Should we update the description as well?

**Comment by andersio** [comment]
=[

**Review by andersio — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chitrakotwani) [reply README.md:76]: I will leave it for you to update. And am merging this now

**Pull request merged**